### PR TITLE
Convert tests to typescript

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md not updated, internal change

This should be a completely noop change. I did convert our test code to async/await though.

### CODE REVIEW

File: src/init/init.ts [hunk 1]
```diff
@@ -230,7 +230,7 @@ function createSelectPrompt(env: YeomanEnvironment): InquirerQuestion {
  * error will be thrown.
  */
 export async function runGenerator(
-    generatorName: string, options: {[name: string]: any}): Promise<void> {
+    generatorName: string, options?: {[name: string]: any}): Promise<void> {
```
Comment (justinfagnani) [inline src/init/init.ts]: Can ignore for now, but I prefer doing `options: {[name: string]: any} = {}` and removing the next line.

File: src/test/integration/lint_test.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+const fixturePath =
+    path.join(__dirname, '../../../src/test/integration/fixtures/');
+
+suite('polymer lint', function() {
+
+  const binPath = path.join(__dirname, '../../../bin/polymer.js');
+
+  this.timeout(2 * 1000);
+
+  test.only('handles a simple correct case', async () => {
```
Comment (stramel) [inline src/test/integration/lint_test.ts]: Should this test be focused?

Comment (rictic) [reply src/test/integration/lint_test.ts]: Good catch! Thank you thank you thank you!

File: src/init/init.ts (hunk 1 above)
Comment (rictic) [reply src/init/init.ts]: +1, fixed since I needed to resolve the `.only` issue too

**Pull request merged**